src/pages/User/index.js

```diff
@@ -65,8 +66,8 @@ export default class User extends Component {
     this.load(nextPage);
   };
 
-  refreshList = () => {
-    this.setState({ refreshing: true });
+  refreshList = async () => {
+    await this.setState({ refreshing: true, stars: [] });
```

Comment:
O `this.setState` não é uma Promise, o `await` não tem efeito aqui.

Reply:
Equívoco meu

Reply:
apesar que, fiz o teste sem e da key duplicada, com o async/await não da. Vi que o setState pode receber um callback também, sendo assim passei o `this.load`  como cb no `this.setState`